### PR TITLE
DEC-556 Add external collection links

### PR DESCRIPTION
**Pull request by rfox2**
What: Links are now working for external collection cards
How: Using the new external url coming from the honeycomb API, added a bit of code that tests for its presence and links to the designated external url

### CODE REVIEW

**Comment by danthewolfe** [comment]
:+1:

File: app/assets/javascripts/components/collection/CollectionCard.jsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ var CollectionCard = React.createClass({
 
   onClick: function(e) {
     e.preventDefault();
-    window.location = this.collectionUrl(this.props.collection);
+    window.open(this.collectionUrl(this.props.collection), '_blank');
```
Comment (jgondron) [inline app/assets/javascripts/components/collection/CollectionCard.jsx]: Do we want it to open a new window for all collections or just external collections?

Comment (rfox2) [inline app/assets/javascripts/components/collection/CollectionCard.jsx]: It seems to make more sense just to have all of them open to a new window, unless there are usability reasons why we shouldn't. It makes it easy for the user to have two or more exhibits open in order to compare / contrast the content.

**Comment by jgondron** [comment]
:+1:

**Pull request merged**